### ISSUE TITLE
fullscreen working every other time on gnome

### ISSUE THREAD

**Issue by Eliyaan** [floooh/sokol#989]
Hi,
The fullscreen parameter is working unreliably on gnome. Sometimes the window shows up in fullscreen but sometimes it does not.
(reproduced it in triangle-sapp.c by adding `.fullscreen = true,` at line 84)

Have a great day!

**Comment by floooh** [comment]
Yeah, I can reproduce in a `GNOME (Wayland)` session but not in a `KDE (Wayland` session.

I guess GNOME needs to fix this on their side, since it works correctly in KDE.

I'll keep the ticket open in case somebody else wants to investigate.

**Comment by floooh** [comment]
Closing this as stale.
